File: PropertyManagement.API/Models/Property.cs
using System.Collections.Generic;

namespace PropertyManagement.API.Models
{
    public class Property
    {
        public int Id { get; set; }
        public string Country { get; set; }
        public string City { get; set; }
        public string Street { get; set; }
        public string PropertyNumber { get; set; }
        public string Description { get; set; }
        public double FlatArea { get; set; }
        public int RoomNumbers { get; set; }
        public int Level { get; set; }
        public User Owner { get; set; }
        public int OwnerId { get; set; }
        public ICollection<Photo> Photos { get; set; }
    }
}

File: PropertyManagement-SPA/src/app/_models/Property.ts
import { Photo } from './Photo';

export interface Property {
    id: number;
    country: string;
    city: string;
    street: string;
    propertyNumber: string;
    description: string;
    flatArea: number;
    roomNumbers: number;
    level: number;
    ownerId: any;
    photoUrl: string;
    photos?: Photo[];
}
